src/prefixes.py

```diff
@@ -69,3 +69,4 @@
 NCBITAXON='NCBITaxon'
 HGNCFAMILY='HGNC.FAMILY'
 PANTHERFAMILY='PANTHER.FAMILY'
+COMPLEXPORTAL='ComplexPortal'
```

Comment:
@cbizon Should I update this to the biolink prefix `MacroMolecular Entity`?

Reply:
I'm not sure I understand.  Macromolecular entity is not a prefix, it's a type.  What we need to do is add ComplexPortal as a prefix to the Macromolecular Machine type in biolink.